src/scripts.js

```diff
@@ -29,5 +30,21 @@ function displayUserData() {
   stepData.innerText = `Your Step Goal: ${user.dailyStepGoal}, Average Step Goal: ${userRepo.calculateAverageStepGoals()}`
 }
 
+function displayHydrationData() {
+  const hydrationTodayData = document.querySelector('.hydration-today-data');
+  const hydrationWeekData = document.querySelector('.hydration-week-data');
+
+  const weeklyData = hydrationRepo.findWeeklyHydration(user.id, '2019/09/21')
+
+  hydrationTodayData.innerText = `Ounces Drank Today: ${hydrationRepo.findDailyHydration(user.id, '2019/09/21')}`;
+  weeklyData.forEach((data) => {
```

Comment:
This is a really clever way of doing it! I'm glad we reached out for advice on a cleaner solution to this display.